### DISCUSSION TITLE
"Add <CONTACT_NAME> to send a message" text overflows if the contact name is long

### DISCUSSION THREAD

**Discussion by smohamedjavid** [status-im/status-legacy#16686] [General]
# Bug Report

In the chat screen of the removed contact, the `Add <CONTACT_NAME> to send a message` text overflows if the contact name is long.

<img src="https://github.com/status-im/status-mobile/assets/19339952/e2cc51d6-7aef-47c0-bb36-fd2cb130cc74" width="300" />

### Expected behaviour

The text should not overflow.

#### Prerequisites: Have at least one mutual contact

1) Open Status
2) Set/Update the nickname of the contact to the longest one
3) Remove them from contacts
4) Enter the chat of that removed user
5) Check the text in the bottom drawer

### Additional Information

- Status Version: nightly [06/08] 
- Operating System: Android, iOS

**Comment by silenteyesoncode** [comment]
@ibrkhalil @smohamedjavid  hi 
Can i work on this issue

**Comment by ibrkhalil** [comment]
I think this should wait until our designers give us some input on how it should look like if the name is too long, WDYT @smohamedjavid

**Comment by smohamedjavid** [comment]
> I think this should wait until our designers give us some input on how it should look like if the name is too long, WDYT @smohamedjavid

Yes @ibrkhalil, It's better to get some input from the design team.

**Comment by smohamedjavid** [comment]
> @ibrkhalil @smohamedjavid hi
> Can i work on this issue

Hey @silenteyesoncode!

We duly appreciate your interest. Unfortunately, this issue needs to go through the design team quickly for the UI behaviour.
Meanwhile, We highly encourage you to pick any other issue. Feel free to go through the [issues](https://www.github.com/status-im/status-mobile/issues). Thanks!

**Comment by silenteyesoncode** [comment]
Hi @smohamedjavid 
Can you help me out to get the issue to select. I mean what tags should I need to look out ?
